src/registration.css

```diff
@@ -12,9 +76,24 @@ footer
 {
     background-color: #D81B60;
     font-family:'Roboto',sans-serif;
+    font-weight: 300;
     border: none;
     color: white;
     text-transform: uppercase;
     padding: 8px 80px;
-    font-size: 12px;
-}
\ No newline at end of file
+    font-size: 14px;
+}
+
+#dateofbirth, #join-reason
+{
+    font-family:'Roboto',sans-serif;
+    font-weight: 100;
```

Comment:
You could also put the `font-weight: 100` rule into a new class and apply that class in the HTML wherever it is needed. That way, you would not need these 2 IDs at all.